### ISSUE TITLE
[Enhancement]: add password-manager-service to the plugs

### ISSUE THREAD

**Issue by V-Gira** [snapcrafters/wire#15]
### What changes would you like?

There is a [password-manager-service interface](https://snapcraft.io/docs/password-manager-service-interface) that could allow the Wire Snap to access the desktop environment's keychain

### Any extra information?

Hi @popey , and happy belated birthday!

I work at [Wire](https://github.com/wireapp), and our [latest release](https://github.com/wireapp/wire-webapp/releases/tag/2024-04-11-production.0) introduced [an issue](https://github.com/wireapp/wire-desktop/issues/7764) for some of our Linux users, most relevant the Snap has become unusable.

We started to use the system keychain to store users' authentication data, relying on Electron's [safeStorage](https://www.electronjs.org/docs/latest/api/safe-storage), and AppArmor policies now prevent users using the Snap  package to log in.

I am not familiar with Snaps, a bit of digging makes me believe we would need to add the `password-manager-service` interface to the list of plugs in the `.yaml` file.

I would be very grateful if you could help me solve this issue, just pointing me in the right direction would be appreciated, I'm perfectly fine submitting a PR.

Thank you for all you're doing,, Linux support for Wire is not yet in the card unfortunately, but I'll be happy to help in anyway I can to render the Snap usable again.

Don't hesitate to reach out, thanks again!

**Comment by V-Gira** [comment]
Unfortunately, I have not been able to build a working Snap following those steps on an Ubuntu VM:

```
snap download wire
```
```
unsquashfs wire_365.snap
```
- edit the `snap.yaml` file in `./squashfs-root/meta` to add `password-manager-service` to the plugs
```
snap pack squashfs-root
```
```
snap install ./wire_3.32.3079_amd64.snap --dangerous
```
The resulting Snap starts properly at that point but suffers from the issue as `password-manager-service` is not connected

after connecting the interface:
```
snap connect wire:password-manager-service
```

The Snap hangs on start up following those steps. I seem to be missing something.

**Comment by V-Gira** [comment]
I was apparently mistaken, the above steps worked perfectly on an up to date system (ubuntu and kubuntu). I'll raise a PR

**Comment by jnsgruk** [comment]
Fixed in #16
